Bio/SeqIO/AbiIO.py

```diff
@@ -203,7 +422,11 @@ def _abi_parse_header(header, handle):
         # only parse desired dirs
         key = _bytes_to_string(dir_entry[0])
         key += str(dir_entry[1])
-        if key in (list(_EXTRACT) + _SPCTAGS):
+
+        global_tag_listing = []
+        for x in _INSTRUMENT_SPECIFIC_TAGS:
+            global_tag_listing += _INSTRUMENT_SPECIFIC_TAGS[x].keys()
+        if key in global_tag_listing:
```

Comment:
Oh jeez, can't believe I didn't catch that. That's what I get for trying to modify everything in place and not really improve upon the code.

I've moved it to the top just after `_INST...` initialization. Thanks, the `.values()` is a good suggestion.